lib/ActivityStreams.js

```diff
@@ -107,13 +109,20 @@ function ActivityStreams(metadataStore, options = {}) {
 
   this._asyncBuildPreviewCache();
   this._startPeriodicBuildPreviewCache(this.options.previewCacheTimeout);
+
+  this._recommendationProvider = new RecommendationProvider(this._previewProvider);
```

Comment:
So, we're instantiating the recommendation provider regardless of whether we will use it or not as implemented.
Can we watch a pref and set it instead?

Should the pref be on, `_asyncGetRecommendations` can be called and `_startRecommendationRefresh` can be invoked if it isn't. since it's a timeout, it's easy to see if it's been set or not